sass/layout/_headers.scss

```diff
@@ -228,6 +243,7 @@
     .heading {
         overflow: hidden;
         margin: 0 0 0 2px;
+        color: var(--center-channel-color);
```

Comment:
Why are we adding these many "new" color all over the components? What is the change that make this required?

Reply:
the change is that it was previously done with the `changeCss` function inside `applyTheme`